src/coreclr/src/jit/lower.cpp

```diff
@@ -2060,10 +2060,9 @@ void Lowering::LowerFastTailCall(GenTreeCall* call)
 
 #else // !FEATURE_FASTTAILCALL
 
-    // Platform choose not to implement fast tail call mechanism.
-    // In such a case we should never be reaching this method as
-    // the expectation is that IsTailCallViaHelper() will always
-    // be true on such a platform.
+    // Platform choose not to implement fast tail call mechanism. In such a
```

Comment:
Nit: Platform choose not to implement fast tail call mechanism. In such a case we should never be reaching this... -> Platform does not implement fast tail call mechanism. This should be unreached because we would choose to do a tailcall via helper.

Thanks for the change, I know you didn't actually modify this line entirely..